substra/sdk/backends/local/backend.py

```diff
@@ -98,8 +98,8 @@ def list_task_input_assets(self, compute_task_key: str) -> List[models.InputAsse
 
         return inputs
 
-    def get_performances(self, key):
-        performances = self._db.get_performances(key)
+    def get_performances(self, compute_plan: models.ComputePlan):
```

Comment:
Nitpicking, but if we are improving signature typing, could you add the return value too?